ste/JobPartPlanFileName.go

```diff
@@ -218,6 +218,7 @@ func (jpfn JobPartPlanFileName) Create(order common.CopyJobPartOrderRequest) {
 		DeleteSnapshotsOption:          order.BlobAttributes.DeleteSnapshotsOption,
 		PermanentDeleteOption:          order.BlobAttributes.PermanentDeleteOption,
 		RehydratePriority:              order.BlobAttributes.RehydratePriority,
+		Metadata:                       order.BlobAttributes.Metadata,
```

Comment:
Pretty sure it's not required.

Reply:
there's no existing way of retrieving metadata for setproperties

Reply:
See how we set metadata. We already persist metadata in plan files. All we need is a way to extract is. I'm sure there are other occurrences in the code.

Reply:
Okay I will. What I was trying to say was that what we do currently is that we have a flow to "preserve metadata" while copying file from one place to another. 
When we're copying from local, the user can define what metadata they want to upload with and we pass it to the common.CopyJobPartOrderRequest{}. However, this struct is passed to the enumerator only in case of the copy command and no other case, not even ours. Hence, I don't see a way of passing that metadata back if we don't do this

Reply:
oh i get it now! You were right. Removing this.